README.md

```diff
@@ -0,0 +1,48 @@
+<div align="center">
+  <a href="https://github.com/fission-codes/fission-cli" target="_blank">
+    <img src="./assets/logo.png" alt="Fission-Cli Logo" width="100"></img>
+  </a>
+
+  <h1 align="center">Fission-Cli</h1>
+
+  <p>
+    <!-- <a href="https://github.com/fission-codes/rust-template/actions?query=">
+      <img src="https://github.com/fission-codes/rust-template/actions/workflows/build.yml/badge.svg" alt="Build Status">
+    </a> -->
+    <a href="./LICENSE">
+      <img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" alt="License-Apache">
+    </a>
+    <a href="https://fission.codes/discord">
+      <img src="https://img.shields.io/static/v1?label=Discord&message=join%20us!&color=mediumslateblue" alt="Discord">
+    </a>
+    <a href="https://talk.fission.codes">
+      <img src="https://img.shields.io/discourse/https/talk.fission.codes/topics" alt="Discourse">
+    </a>
+  </p>
+</div>
+
+##
+
+### In Progress Notice
+This repository is a rewrite of our current Cli that can be found [here](https://github.com/fission-codes/fission/tree/main/fission-cli). The progress we are making tword finishing the rewrite can be seen on [this tracking issue](https://github.com/fission-codes/fission-cli/issues/1). However, our plan is to wrap the existing binary so that the application be used (hopefully) seemlessly even while the rewrite is still in progress.
```

Comment:
```suggestion
This repository is a rewrite of our current CLI that can be found [here](https://github.com/fission-codes/fission/tree/main/fission-cli). The progress we are making toward finishing the rewrite can be seen in [this tracking issue](https://github.com/fission-codes/fission-cli/issues/1). However, our plan is to wrap the existing binary so that the application be used even while the rewrite is still in progress.
```

A few copy edits and suggestions.